wandb/sdk/wandb_settings.py

```diff
@@ -515,6 +519,7 @@ def _default_props(self) -> Dict[str, Dict[str, Any]]:
             run_tags={
                 "preprocessor": lambda x: tuple(x) if not isinstance(x, tuple) else x,
             },
+            run_url={"value": "<run_url>", "hook": lambda x: self._get_run_url(),},
```

Comment:
could you also add support of `sweep_url` and `project_url`?

Reply:
@dmitryduev and @vwrj when are you adding new objects as `@property` vs adding it here?

Reply:
Based on what's available when the Settings object is initialized. These are run-time hooks because entity and project aren't available when Settings is init. @dmitryduev

Reply:
Yep, I can add `sweep_url` and `project_url`.

Reply:
Hmm... That's a great point @kptkin! I'd say:
- For any prop that is only computed from other props and so does not require any assignment validation etc., make it a @property
- For everything else, make it a Property object and define in Settings (class attributes + optionally default props).

Reply:
@vwrj mind refactoring it to be a @property? [It would have been useful to make it a Property object if the hook actually depended on x, which is not the case here]